src/comonents/InputForm.jsx

```diff
@@ -1,11 +1,21 @@
-import React from 'react'
+import React, { useContext } from "react";
+import ButtonPage from "./ButtonPage";
+import { MyNoteContext } from "../context/NoteState";
 
 function InputForm() {
+  const { todo, handleChange } = useContext(MyNoteContext);
   return (
-    <div>
-      
+    <div style={{ width: "100%", display: "flex" }}>
```

Comment:
remove inline css, create separate file for css and attach

Reply:
Resolved sir